egs2/mgb2/asr1/local/xml2stm.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
```

Comment:
Umm.. I guess most of the files are from Kaldi right? If possible, you can directly make a softlink (just as we did for steps and utils). It turns out a bit difficult to maintain those files (and keep syncing with Kaldi) on our own. Since we already would have a git-cloned Kaldi, I think the soft link should be fine.